### DISCUSSION TITLE
Numerical Instability Error

### DISCUSSION THREAD

**Discussion by Clara-Marianne** [firemodels/fds#12876] [Q&A] [answered]
Hello everyone,

I am currently conducting a simulation lasting 30 seconds. I have implemented both an LSP (Laminar Smoke Point) soot model and a MOSS model in FDS. I performed the simulation with both models using two cases: 1- pyrolysis and 2- imposition of experimentally found mass loss. When the mass loss is imposed, both soot models provide results. However, with the pyrolysis model, only the LSP model allows me to successfully complete the simulation. Unfortunately, with the MOSS model, I encounter a numerical instability error after 14 seconds. I have attempted to resolve this issue several times and have referred to the user guide, specifically the section "Common Error Statements > Numerical Instability Errors", but despite my efforts, I have been unable to identify the source of this instability. Can you please help me have a look?

&HEAD CHID='heptane',TITLE='Liquid Fire - C7H16' /

&MESH IJK=100,10,150, XB = -0.5,0.5,-0.5,-0.4,0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5,-0.4,-0.3,0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5,-0.3,-0.2,0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5,-0.2,-0.1,0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5,-0.1, 0.0,0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5, 0.0,0.1, 0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5, 0.1,0.2, 0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5, 0.2,0.3, 0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5, 0.3,0.4, 0.,1.5/
&MESH IJK=100,10,150, XB = -0.5,0.5, 0.4,0.5, 0.,1.5/

&TIME T_END=30. /
&DUMP SMOKE3D=.TRUE.,DT_RESTART=1.0/
&COMB RADIATIVE_FRACTION=0., EXTINCTION_MODEL='EXTINCTION 2'/ 

&MISC TMPA=25., DNS=.FALSE.,TURBULENCE_MODEL='DEARDORFF',SURF_DEFAULT='STEEL_WALL',RESTART=.FALSE.,
GRAVITATIONAL_DEPOSITION=.TRUE.,THERMOPHORETIC_DEPOSITION=.TRUE.,TURBULENT_DEPOSITION=.TRUE.,GRAVITATIONAL_SETTLING=.TRUE.,THERMOPHORETIC_SETTLING=.TRUE./ 

&RADI WIDE_BAND_MODEL=.FALSE./

&SPEC ID='N-HEPTANE', MASS_FRACTION_0=0.0 /
&SPEC ID='NITROGEN', BACKGROUND=.TRUE./
&SPEC ID='OXYGEN', MASS_FRACTION_0 = 0.23 /
&SPEC ID='CARBON MONOXIDE', MASS_FRACTION_0 = 0.0 /
&SPEC ID='WATER VAPOR', MASS_FRACTION_0=0.0/
&SPEC ID='CARBON DIOXIDE', MASS_FRACTION_0 = 0.0 /
&SPEC ID='SOOT', MASS_FRACTION_0 = 0.0, AEROSOL=.TRUE., 
          MEAN_DIAMETER=2.E-8, FORMULA='C' /
&SPEC ID='HYDROGEN', MASS_FRACTION_0 = 0.0 /

!!! DEFINITION DES REACTIONS !!!

&REAC ID = 'R1'
      FUEL = 'N-HEPTANE'
      SPEC_ID_NU='N-HEPTANE','OXYGEN','CARBON DIOXIDE','WATER VAPOR','CARBON MONOXIDE','HYDROGEN'
      NU= -1,-10,6,7,1,1
      CRITICAL_FLAME_TEMPERATURE=867./

! A in R2 is tuned to match Smyth CO data

&REAC ID = 'R2'
      FUEL = 'CARBON MONOXIDE'
      A = 1.5e9
      E = 4.1840e4
      SPEC_ID_NU = 'CARBON MONOXIDE','OXYGEN','CARBON DIOXIDE'
      NU = -1,-0.5,1
      SPEC_ID_N_S = 'OXYGEN','CARBON MONOXIDE','WATER VAPOR'
      N_S = 0.25,1,0.5 /

&REAC ID = 'R3'
      FUEL = 'SOOT'
      A = 4.7e10
      E = 2.11e5
      SPEC_ID_NU= 'SOOT','OXYGEN','CARBON DIOXIDE'
      NU= -1,-1,1
      SPEC_ID_N_S='SOOT','OXYGEN'
      N_S = 1,1
      N_T = 0.0/

&REAC ID = 'R4'
      FUEL = 'CARBON DIOXIDE'
      A = 6.16e13
      E = 3.28026e5
      SPEC_ID_NU= 'CARBON DIOXIDE','OXYGEN','CARBON MONOXIDE'
      NU= -1,0.5,1
      SPEC_ID_N_S='OXYGEN','CARBON DIOXIDE','WATER VAPOR'
      N_S = -0.25,1,0.5
      N_T = -0.97/

&REAC ID = 'R5'
	REVERSE = T
	FUEL = 'HYDROGEN'
	A = 0.85e16
	E = 167360.
	RADIATIVE_FRACTION=0.0
	SPEC_ID_NU='HYDROGEN','OXYGEN','WATER VAPOR'
	SPEC_ID_N_S='HYDROGEN','OXYGEN'
	NU= -1,-0.5,1
	N_S = 0.25,1.5
	N_T = -1 /
	

&SURF ID                 = 'STEEL_WALL'
       MATL_ID           = 'STEEL'
       BACKING           = 'INSULATED'
       THICKNESS         = 0.006 
       COLOR             = 'GRAY' /  

&MATL ID='STEEL', SPECIFIC_HEAT=0.770, CONDUCTIVITY=45.8, EMISSIVITY=0.7, DENSITY=7700. /

&MATL    ID = 'WATER'
       CONDUCTIVITY = 0.60
      EMISSIVITY   = 0.96
      SPECIFIC_HEAT = 4.19
      DENSITY = 1000. /

&SURF             ID = 'WATER_POOL'
                  MATL_ID = 'WATER'
                  COLOR = 'RED'
                  THICKNESS = 0.5/

&MATL ID                     = 'HEPTANE LIQUID'
      EMISSIVITY             = 0.9
      NU_SPEC                = 1.
      SPEC_ID                = 'N-HEPTANE'
      HEAT_OF_REACTION       = 364.9
      CONDUCTIVITY           = 0.14
      SPECIFIC_HEAT          = 2.2464
      BOILING_TEMPERATURE    = 98.5       
      DENSITY                = 684.
      ABSORPTION_COEFFICIENT = 250. / 

&SURF ID        = 'HEPTANE POOL'
      COLOR     = 'RASPBERRY'
      MATL_ID   = 'HEPTANE LIQUID'
      THICKNESS = 0.05 
      CONVECTION_LENGTH_SCALE=1.0 
      EXTERNAL_FLUX = 20.
      EXTERNAL_FLUX_RAMP = 'preheating'
      BACKING='INSULATED' /
      
&RAMP ID= 'preheating', T= 0.	, F=1.0/
&RAMP ID= 'preheating', T= 1.	, F=1.0/
&RAMP ID= 'preheating', T= 2.	, F=0.0/

&SURF ID='BURNER', COLOR='YELLOW', MASS_FLUX(1)=0.033, SPEC_ID(1)='N-HEPTANE',TMP_FRONT=150./

&OBST XB=-0.15,0.15,-0.15,0.15,0.08,0.1,SURF_IDS='HEPTANE POOL','STEEL_WALL','WATER_POOL'  / FIRE POOL
&OBST XB=-0.15,0.15,-0.15,0.15,0.0,0.08,SURF_IDS='WATER_POOL','STEEL_WALL','STEEL_WALL'  / Water POOL
&VENT XB=-0.15,0.15,-0.15,0.15,0.1,0.1,XYZ=0,0,0, RADIUS=0.15, SURF_ID='HEPTANE POOL'/

&VENT MB='XMIN', SURF_ID='OPEN' / 
&VENT MB='XMAX', SURF_ID='OPEN' / 
&VENT MB='YMIN', SURF_ID='OPEN' / 
&VENT MB='YMAX', SURF_ID='OPEN' / 
&VENT MB='ZMAX', SURF_ID='OPEN' /

&SLCF PBY=0.0,QUANTITY='VELOCITY', VECTOR=.TRUE. /
&SLCF PBY=0.0,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.0,QUANTITY='HRRPUV', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.0,QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.0,QUANTITY='VOLUME FRACTION', SPEC_ID='CARBON MONOXIDE', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.0,QUANTITY='VOLUME FRACTION', SPEC_ID='CARBON DIOXIDE', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.0,QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.0,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE. /

&SLCF XB=-0.3,0.3,0.0,0.0,0.07,0.07,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE. /
&SLCF XB=-0.3,0.3,0.0,0.0,0.46,0.46,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE. /
&SLCF XB=-0.3,0.3,0.0,0.0,1.03,1.03,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE. /
&SLCF XB=-0.3,0.3,0.0,0.0,1.27,1.27,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE. /

&SLCF XB=-0.3,0.3,0.0,0.0,0.07,0.07,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF XB=-0.3,0.3,0.0,0.0,0.46,0.46,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF XB=-0.3,0.3,0.0,0.0,1.03,1.03,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF XB=-0.3,0.3,0.0,0.0,1.27,1.27,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /

&DEVC XYZ=0.00,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h0.07'/
&DEVC XYZ=0.03,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h0.07'/
&DEVC XYZ=0.06,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h0.07'/
&DEVC XYZ=0.09,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h0.07'/
&DEVC XYZ=0.12,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h0.07'/
&DEVC XYZ=0.15,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h0.07'/

&DEVC XYZ=0.00,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h0.46' /
&DEVC XYZ=0.03,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h0.46' /
&DEVC XYZ=0.06,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h0.46' /
&DEVC XYZ=0.09,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h0.46' /
&DEVC XYZ=0.12,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h0.46' /
&DEVC XYZ=0.15,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h0.46' /

&DEVC XYZ=0.00,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h1.03'/
&DEVC XYZ=0.03,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h1.03'/
&DEVC XYZ=0.06,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h1.03'/
&DEVC XYZ=0.09,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h1.03'/
&DEVC XYZ=0.12,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h1.03'/
&DEVC XYZ=0.15,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h1.03'/

&DEVC XYZ=0.00,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h1.27'/
&DEVC XYZ=0.03,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h1.27'/
&DEVC XYZ=0.06,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h1.27'/
&DEVC XYZ=0.09,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h1.27'/
&DEVC XYZ=0.12,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h1.27'/
&DEVC XYZ=0.15,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h1.27'/

&DEVC XYZ=0.00,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.03,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.06,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.09,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.12,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.15,0.,0.07,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h0.07RMS',Statistics='RMS'/

&DEVC XYZ=0.00,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.03,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.06,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.09,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.12,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.15,0.,0.46,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h0.6RMS' ,Statistics='RMS'/

&DEVC XYZ=0.00,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.03,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.06,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.09,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.12,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.15,0.,1.03,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h1.03RMS',Statistics='RMS'/

&DEVC XYZ=0.00,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R0h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.03,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R1h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.06,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R2h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.09,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R3h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.12,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R4h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.15,0.,1.27,        QUANTITY='AEROSOL VOLUME FRACTION',  ID='SOOT_R5h1.27RMS',Statistics='RMS'/

&DEVC XYZ=0.00,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h0.07'/
&DEVC XYZ=0.03,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h0.07'/
&DEVC XYZ=0.06,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h0.07'/
&DEVC XYZ=0.09,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h0.07'/
&DEVC XYZ=0.12,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h0.07'/
&DEVC XYZ=0.15,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h0.07'/

&DEVC XYZ=0.00,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h0.46' /
&DEVC XYZ=0.03,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h0.46' /
&DEVC XYZ=0.06,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h0.46' /
&DEVC XYZ=0.09,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h0.46' /
&DEVC XYZ=0.12,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h0.46' /
&DEVC XYZ=0.15,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h0.46' /

&DEVC XYZ=0.00,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h1.03'/
&DEVC XYZ=0.03,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h1.03'/
&DEVC XYZ=0.06,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h1.03'/
&DEVC XYZ=0.09,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h1.03'/
&DEVC XYZ=0.12,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h1.03'/
&DEVC XYZ=0.15,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h1.03'/

&DEVC XYZ=0.00,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h1.27'/
&DEVC XYZ=0.03,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h1.27'/
&DEVC XYZ=0.06,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h1.27'/
&DEVC XYZ=0.09,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h1.27'/
&DEVC XYZ=0.12,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h1.27'/
&DEVC XYZ=0.15,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h1.27'/

&DEVC XYZ=0.00,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.03,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.06,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.09,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.12,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h0.07RMS',Statistics='RMS'/
&DEVC XYZ=0.15,0.,0.07,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h0.07RMS',Statistics='RMS'/

&DEVC XYZ=0.00,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.03,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.06,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.09,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.12,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h0.6RMS' ,Statistics='RMS'/
&DEVC XYZ=0.15,0.,0.46,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h0.6RMS' ,Statistics='RMS'/

&DEVC XYZ=0.00,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.03,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.06,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.09,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.12,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h1.03RMS',Statistics='RMS'/
&DEVC XYZ=0.15,0.,1.03,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h1.17RMS',Statistics='RMS'/

&DEVC XYZ=0.00,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R0h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.03,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R1h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.06,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R2h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.09,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R3h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.12,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R4h1.27RMS',Statistics='RMS'/
&DEVC XYZ=0.15,0.,1.27,        QUANTITY='TEMPERATURE',  ID='TEMP_R5h1.27RMS',Statistics='RMS'/

!&DEVC XB=-0.3,0.3,0.0,0.0,0.06,0.06,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE., STATISTICS='RMS' /
!&DEVC XB=-0.3,0.3,0.0,0.0,0.45,0.45,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE., STATISTICS='RMS' /
!&DEVC XB=-0.3,0.3,0.0,0.0,1.02,1.02,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE., STATISTICS='RMS' /
!&DEVC XB=-0.3,0.3,0.0,0.0,1.26,1.26,QUANTITY='AEROSOL VOLUME FRACTION', SPEC_ID='SOOT', CELL_CENTERED=.TRUE., STATISTICS='RMS' /

!&DEVC XB=-0.3,0.3,0.0,0.0,0.06,0.06,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., STATISTICS='RMS' /
!&DEVC XB=-0.3,0.3,0.0,0.0,0.45,0.45,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., STATISTICS='RMS' /
!&DEVC XB=-0.3,0.3,0.0,0.0,1.02,1.02,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., STATISTICS='RMS' /
!&DEVC XB=-0.3,0.3,0.0,0.0,1.26,1.26,QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., STATISTICS='RMS' /

&BNDF QUANTITY='RADIATIVE HEAT FLUX' /
&BNDF QUANTITY='CONVECTIVE_FLUX' /
&BNDF QUANTITY='BURNING_RATE' / 
&BNDF QUANTITY='WALL_TEMPERATURE' / 
&BNDF QUANTITY='SURFACE DEPOSITION', SPEC_ID='SOOT'/
&TAIL /

**Comment by Er9y714** [comment]
Are you using the latest version? FDS 6.9.1

**Reply by Clara-Marianne** [reply]
No, I'm using version 6.7.9 of FDS

**Comment by drjfloyd** [comment, marked as answer]
When you say you have implemented a model, does that mean you changed the source code or this is just something you are doing with your inputs?

**Reply by Clara-Marianne** [reply]
Yes, I modified a few lines of the soot.f90 file before compiling FDS.

**Comment by mcgratta** [comment]
Some of these parameters
```
&MISC TMPA=25., DNS=.FALSE.,TURBULENCE_MODEL='DEARDORFF',SURF_DEFAULT='STEEL_WALL',RESTART=.FALSE.,
GRAVITATIONAL_DEPOSITION=.TRUE.,THERMOPHORETIC_DEPOSITION=.TRUE.,TURBULENT_DEPOSITION=.TRUE.,GRAVITATIONAL_SETTLING=.TRUE.,THERMOPHORETIC_SETTLING=.TRUE./
```
are no longer used in the most recent version. Use the latest version of FDS and bring the input file up to date.

**Reply by Clara-Marianne** [reply]
Alright, I'll do that.
